src/components/EntityInfo.vue

```diff
@@ -108,20 +108,34 @@
 
         <v-col cols="12" md="3">
           <dl>
+
+            <!-- Registration Date -->
+            <template v-if="businessId && isFirm">
+              <dt class="mr-2">Registration Date:</dt>
+              <dd id="entity-business-registration-date">
+                {{ this.dateToPacificDate(getEntityFoundingDate) || 'Not Available' }}</dd>
+            </template>
+
+            <!-- Registration Number -->$
+            <template v-if="businessId && isFirm">
+              <dt class="mr-2">Registration Number:</dt>
+              <dd id="entity-business-registration-number">{{ getIdentifier || 'Not Available' }}</dd>
+            </template>
+
             <!-- Business Number -->
             <template v-if="businessId">
               <dt class="mr-2">Business Number:</dt>
               <dd id="entity-business-number">{{ getBusinessNumber || 'Not Available' }}</dd>
             </template>
 
             <!-- Incorporation Number -->
-            <template v-if="businessId">
+            <template v-if="businessId && !isFirm">
               <dt class="mr-2">Incorporation Number:</dt>
               <dd id="entity-incorporation-number">{{ getIdentifier || 'Not Available' }}</dd>
             </template>
 
             <!-- NR Number -->
-            <template v-if="getNrNumber">
+            <template v-if="getNrNumber && !isFirm">
```

Comment:
I think this code is to display the NR number when a user is registering the firm (in other words, the entity has not been filed and does not have a business ID yet; it is using a temporary registration number).

Reply:
If this is not clear then I can provide some examples.

Reply:
I see, so as long as there is an NR number, it should be displayed?

Reply:
Actually, I confused "NR number" with "temporary registration number". Sorry!

Here, we are looking at the "NR number". I believe this should only be displayed when the user is still registering/incorporating their business (ie, while this page uses temp reg number to load data instead of the permanent entity id). Once the business is registered/incorporated then we don't care about the NR number.

And I believe the NR number is only available before this business is registered/incorporated.

Therefore, I think the old logic is fine: just display it if it's available. (And it should display for all registrations/incorporations with a NR, not just firms.) Let me know if you have more info on this.